### DISCUSSION TITLE
Making a Website

### DISCUSSION THREAD

**Discussion by Bugreporter48** [awesome-jellyfin/awesome-jellyfin#45] [Ideas]
I was considering making a website with all the plugins, themes, custom scripts 

thoughts?

**Comment by JonCastaway** [comment]
How about GitHub pages?

https://pages.github.com/

**Reply by jade-city2** [reply]
I started to make a basic one, however I wasn't sure what to do with clients.md since it's generated.
[https://github.com/jade-city2/unofficial-awesome-jellyfin-website](https://github.com/jade-city2/unofficial-awesome-jellyfin-website)
